spa/src/common/Wrapper.tsx

```diff
@@ -1,14 +1,15 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
Is there any way we can implement `css` without adding this in header for every file?

Reply:
This got removed

Reply:
We should be able to implement this globally using webpack @krazziekay. Wanna discuss via zoom?

Reply:
Yep
@kamaksheeAtl  can you please setup some time today or tomorrow?

Reply:
sure